SonarQube.MSBuild.Tasks/AttachBuildWrapper.cs

```diff
@@ -117,6 +115,31 @@ private bool CheckRequiredFilesExist()
         }
 
         /// <summary>
+        /// Tries to get the location to which the build output files should be written from the config file
+        /// </summary>
+        private bool TryGetOutputDirectory(out string outputDir)
+        {
+            outputDir = null;
+
+            AnalysisConfig config = TaskUtilities.TryGetConfig(this.AnalysisConfigDir, new MSBuildLoggerAdapter(this.Log));
```

Comment:
Re-uses the _TaskUtilities_ helper method that is also used by the _IsTestByName_ task.